magic_spoiler/__main__.py

```diff
@@ -208,15 +208,16 @@ def open_header(card_xml_file: IO[Any]) -> None:
     :param card_xml_file: Card file path
     """
     card_xml_file.write(
-        "<cockatrice_carddatabase version='4'>\n"
-        + "  <!--\n"
-        + "  Created At: "
-        + datetime.datetime.utcnow().strftime("%a, %b %d %Y, %H:%M:%S")
-        + " (UTC)\n"
-        + "  Created By: Magic-Spoiler project @ https://github.com/Cockatrice/Magic-Spoiler\n"
-        + "  \n"
-        + "  THIS FILE IS AUTOMATICALLY GENERATED & ALL EDITS WILL BE OVERRIDDEN.\n"
-        + "  -->\n"
+        "<!--\n"
+        + "THIS FILE IS AUTOMATICALLY GENERATED ON A REGULAR BASIS. ALL MANUAL EDITS WILL BE OVERRIDDEN.\n"
+        + "-->\n"
+        * "<cockatrice_carddatabase version='4' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='https://raw.githubusercontent.com/Cockatrice/Cockatrice/master/doc/carddatabase_v4/cards.xsd'>\n"
+        + "<info>\n"
+        + "  <author>Cockatrice/Magic-Spoiler</author>\n"
+        + "  <createdAt>" + datetime.datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S") + " (UTC)" + "</createdAt>\n"
+        + "  <sourceUrl>https://github.com/Cockatrice/Magic-Spoiler/blob/files/spoiler.xml</sourceUrl>\n"
+        + "  <sourceVersion>" + datetime.datetime.utcnow().strftime("%Y%m%d") + "</sourceVersion>\n"
```

Comment:
this version is irrelevant, and useless when updated multiple times a day

Reply:
From the PR desciption:

>Version output used is the same as in the https://github.com/Cockatrice/Magic-Token repo: `YYYYMMDD`, even though I'm not a great fan of it.
Since we do it manually over there we can append incrementing alphabetical letters if there are changes and various versions for the same day. Here, we would need to append e.g. the time `YYYYMMDD+HHMMSS` if we want to avoid additional logic.
It would be good to have it consistent and automated in all places we control.

Reply:
I'm about to open an issue in the token repo to finally automate it there as well. The version bumps are always missed.
Probably something that you might need to tackle... 😉 

Which pattern do you think is most reasonable if we stick with the date for now?
`YYYYMMDD+HHMMSS` or the appended letters `YYYYMMDDa`? Could also be numbers I guess `YYYYMMDD_x`?

Reply:
either use the git hash or just leave it unused

Reply:
```suggestion
        + "  <sourceVersion></sourceVersion>\n"
```